app/models/contract.rb

```diff
@@ -0,0 +1,7 @@
+class Contract < ApplicationRecord
+  belongs_to :offer
+
+  def get_deadline
+    return self[:created_at] + (2*7*24*60*60)
```

Comment:
No need for explicit `return` when the value we want to return is the expression on the last line of the function